src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb

```diff
@@ -269,14 +373,21 @@
     "\n",
     "[Explore English Premier League using SPARQL](../02-Visualization/EPL-SPARQL.ipynb)\n",
     "\n",
+    "[Explore English Premier League using openCypher](../02-Visualization/EPL-openCypher.ipynb)\n",
+    "\n",
     "Curious about the business problems can be solved with graph? Check out these sample application notebooks for some inspiration.\n",
     "\n",
     "[Introduction to Fraud Graphs](../03-Sample-Applications/01-Fraud-Graphs/01-Building-a-Fraud-Graph-Application.ipynb)\n",
     "\n",
-    "[Introduction to Knowledge Graphs](../03-Sample-Applications/02-Knowledge-Graphs/01-Building-a-Knowledge-Graph-Application.ipynb)\n",
+    "[Introduction to Knowledge Graphs](../03-Sample-Applications/02-Knowledge-Graphs/01-Building-a-Knowledge-Graph-Application-Gremlin.ipynb)\n",
     "\n",
-    "[Introduction to Identity Graphs](../03-Sample-Applications/03-Identity-Graphs/01-Building-an-Identity-Graph-Application.ipynb)"
-   ]
+    "[Introduction to Identity Graphs](../03-Sample-Applications/03-Identity-Graphs/01-Building-an-Identity-Graph-Application.ipynb)\n",
+    "\n",
+    "[Introduction to Security Graphs](../03-Sample-Applications/04-Security-Graphs/01-Building-a-Security-Graph-Application-with-Gremlin.ipynb)"
```

Comment:
Same with this link, update to the higher level so users can choose based on language: https://github.com/aws/graph-notebook/tree/main/src/graph_notebook/notebooks/03-Sample-Applications/04-Security-Graphs

Reply:
Done